.github/workflows/builduploads3.yml

```diff
@@ -0,0 +1,41 @@
+# This workflow uploads the latest wheel of hydra-core & ray-launcher to S3 bucket.
```

Comment:
Maybe you should switch to a locally initiated solution for now?
The added benefit of this as far as I can tell is that you don't need to initial a manual push. on the other hand it requires you to push the code to github so it's not so different.